README.md

```diff
@@ -14,6 +14,56 @@ not yet suited for any sort of production use.
 
 ### Prerequisites
 
+There are two separate methods for building the bootstrap compiler (Docker and Native). Follow
+prerequisites for your preferred method.
+
+#### (Preferred) Docker Prerequisites
```

Comment:
`(Preferred)` here is confusing. I first thought it meant that these are the preferred docker-specific instructions, but that there's an alternative docker approach.

Reply:
```suggestion
#### Docker Prerequisites
```